var/spack/repos/builtin/packages/ams/package.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Ams(CMakePackage, CudaPackage):
+    """AMS Autonomous Multiscale Framework."""
+
+    homepage = "https://github.com/LLNL/AMS"
+    git = "git@github.com:LLNL/AMS.git"
+
+    maintainers = ["koparasy", "lpottier"]
+
+    version("07.25.23-alpha", tag="07.25.23-alpha", submodules=False)
+    version("develop", branch="develop", submodules=False)
```

Comment:
Please move the branch version to above the tagged one.  It is considered "newer" than numbered versions (see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison).

Reply:
Fixed